deploy.sh

```diff
@@ -210,15 +210,22 @@ if [ "$deploy_process_type" = "scheduledtasks" ]; then
 	echo "----> Deploying scheduled tasks as EventBridge rules"
 	deploy_scheduled_tasks_path=$(grep scheduledtasks Procfile | cut -d':' -f2 | xargs)
 	if [ -f "$deploy_scheduled_tasks_path" ]; then
-        deploy_alb_subnets=$(jq --raw-input --raw-output 'split(",")' <<<"$ECS_SERVICE_SUBNETS")
-        deploy_alb_security_groups=$(jq --raw-input --raw-output 'split(",")' <<<"$ECS_SERVICE_SECURITY_GROUPS")
+		deploy_alb_subnets=$(jq --raw-input --raw-output 'split(",")' <<<"$ECS_SERVICE_SUBNETS")
+		deploy_alb_security_groups=$(jq --raw-input --raw-output 'split(",")' <<<"$ECS_SERVICE_SECURITY_GROUPS")
+
 		while IFS= read -r line; do
-			deploy_scheduled_task_name=$(echo $line | cut -d' ' -f1)
+			if [ -z $line ]; then
+				break
+			fi
+    	# Use $line value literally
+			line=$(echo "$line")
```

Comment:
I didn't get the point of this very line.

Reply:
Really unnecessary. Tested and removed from PR.